bigunko.show

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+DIR="$(cd "$(dirname "${BASH_SOURCE:-$0}")";pwd)"
+
+echo -e $(cat $DIR/bigunko.txt|base64 -d)
```

Comment:
Why don't you use `echo -ne` ?
It can be much more beautiful UNKO 😄 

![screen shot 2017-08-29 at 11 20 56 pm](https://user-images.githubusercontent.com/3874767/29825914-c9d900c6-8d10-11e7-9a66-655fbba6506d.png)

Reply:
Sorry, this is caused by my unique terminal settings.
Never mind.